Server/sprawl_calculator/wup_calculator/tasks.py

```diff
@@ -0,0 +1,70 @@
+from celery import Celery
+from urban_sprawl.raster_helper.raster_downloader import download_raster
+from urban_sprawl.raster_helper.raster_operations import clip_raster_with_geojson
+from urban_sprawl.dis.dis_calculator import DisCalculator
+from urban_sprawl.common.common import Common
+from urban_sprawl.raster_helper.data_definitions import GeoJSONFeature
+from .data_definitions import WUPCalculationResult
+import rasterio
+from urban_sprawl.wup_processing import (
+    get_si_raster_server,
+    calculate_and_save_dis,
+    save_raster,
+    get_si_lib,
+    get_output_path,
+    calculate_and_save_wdis,
+    calculate,
+)
+from osgeo import gdal
+
+app = Celery('tasks', broker='redis://localhost:8000')
+
+@app.task
+def process_algorithm_async(url:str, geojson:GeoJSONFeature):
```

Comment:
in a celery task the parameters must be a string then re-serialised so the data type for Geojson is a sting

Reply:
A 'algorithm parameter' object must be constructed see the sync view.